tools/build_config/Jenkinsfile

```diff
@@ -182,10 +190,24 @@ pipeline {
     label env.AGENT
   }
 
+  environment {
+    MATLAB_VERSION = get_param('MATLAB_VERSION', get_matlab_release(env.JOB_BASE_NAME))
```

Comment:
a comment here or in the get_param function would make it explicit that this will use the variable passed from Jenkins UI OR the default if that is empty.